### DISCUSSION TITLE
Prometheus Exporter: Prefix

### DISCUSSION THREAD

**Discussion by alydharshi** [openconfig/gnmi-gateway#33] [Ideas]
Hello Devs,

I was wondering if there was a way to add a prefix to the prometheus output something like say gnmi_<remaining string> instead of simply passing out the gnmi output. This can help differentiate and quickly allow us for example to find values in Grafana or other such visualization tools. Thanks.

Cheers,

ASD.

**Comment by colinmcintosh** [comment]
This is not currently available in the Prometheus Exporter but I do a similar thing with our internal metrics exporters so I don't see a reason why this couldn't be added to the Prometheus Exporter. Feel free to open a PR adding it in or I'll add it to my to-do list for when I have a chance to get to it.
